frontend/services/sources.go

```diff
@@ -394,3 +416,22 @@ func ToggleSourceCRD(ctx context.Context, nsName string, workloadName string, wo
 		return DeleteSourceCRD(ctx, nsName, workloadName, workloadKind)
 	}
 }
+
+// TODO: remove this after a fix was made in the backend to correctly handle the InstrumentedApplication on-create Source CRD
+func SetWorkloadInstrumentationLabel(ctx context.Context, nsName string, workloadName string, workloadKind WorkloadKind, enabled *bool) error {
```

Comment:
we already have this func in the utils.go

Reply:
It was moved from `utils.go`